pkg/ddl/ingest/env.go

```diff
@@ -22,14 +22,14 @@ import (
 	"github.com/pingcap/tidb/pkg/config"
 	"github.com/pingcap/tidb/pkg/lightning/log"
 	"github.com/pingcap/tidb/pkg/util"
-	"github.com/pingcap/tidb/pkg/util/logutil"
 	"github.com/pingcap/tidb/pkg/util/memory"
 	"github.com/pingcap/tidb/pkg/util/size"
 	"go.uber.org/zap"
 )
 
 var (
-	litLogger = logutil.BgLogger().With(zap.String(logutil.LogFieldCategory, "ddl-ingest"))
+	// LitLogger is the package-level logger.
+	LitLogger = zap.NewNop()
```

Comment:
leave previous value? so we can see logs in test

Reply:
In fact I think printing all message of passed unit test is useless, makes me hard to locate the log of failed tests. I'll try to initialize these logger to https://pkg.go.dev/go.uber.org/zap/zaptest#NewLogger

Reply:
> > leave previous value? so we can see logs in test
> 
> In fact I think printing all message of passed unit test is useless, makes me hard to locate the log of failed tests. I'll try to initialize these logger to https://pkg.go.dev/go.uber.org/zap/zaptest#NewLogger

I'll revert to previous values, it's hard to write a helper function and use it once to get `testing.T` for `zaptest.NewLogger`

Reply:
I found https://github.com/pingcap/tidb/blob/master/pkg/statistics/handle/logutil/logutil.go 

Do you think this style is good? The `SamplerLogger` can also replace our manually down-sample logging logic

https://github.com/pingcap/tidb/blob/e8c397ff93ff55da30e553b77bd1a25f6b841510/pkg/ddl/backfilling.go#L279-L280

cc @D3Hunter @zimulala

Reply:
> > leave previous value? so we can see logs in test
> 
> In fact I think printing all message of passed unit test is useless, makes me hard to locate the log of failed tests. I'll try to initialize these logger to https://pkg.go.dev/go.uber.org/zap/zaptest#NewLogger

if the test is flaky, it might help, especially for cases that only trigger on some timeline, such as DXF

Reply:
`zaptest.NewLogger` will print the content when test failed, or use `go test -v`.